### PR TITLE
(Hopefully) corrected some codestyle errors and updated ruff.toml

### PR DESCRIPTION
**Pull request by Bryce-3D**
Fixed some of the checkstyle errors detected by ruff.

Most of the style errors that could be fixed automatically were done. Three of these were excluded however, specifically `D203`, `D212`, and `UP015`, since they seemed unnecessary and `ruff.toml` was updated accordingly.

Imports for unnecessary type hints such as `typing.Dict`, `typing.List`, and `typing.Set` were manually removed.

### CODE REVIEW

File: ruff.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ target-version = "py310"
 [lint]
 select = ["ALL"]
 
-ignore = []
+ignore = ["D203","D212","UP015"]
```
Comment (lunaflight) [inline ruff.toml:7]: What do you think about having a space after each comma?

File: src/history/README.md
Comment (lunaflight) [inline src/history/README.md:1]: The documentation shouldn't have been removed - maybe check that you have the latest version of the repo and merge your PR with the latest version of the repo?

File: src/history/build.py
Comment (lunaflight) [inline src/history/build.py:1]: What do you think about setting `check` to `True` instead? This way, when the shell script fails, the entire function should terminate - it does not make sense to try, for example, importing CSV files to a database if it wasn't able to clean the CSVs in the first place.

**Review by lunaflight — CHANGES_REQUESTED** [review]
Just a few comments, otherwise, looks good to me!

I noticed that you created another PR and closed the previous one - I would prefer if you opened only one PR and just push changes to your current branch. Your pull request will automatically be updated with the latest commits.

File: ruff.toml (hunk 1 above)
Comment (Bryce-3D) [reply ruff.toml:7]: I'm not sure if ruff has some codestyle for this, but I usually don't put spaces for something like this. 
I'm fine with or without though so your choice.

File: ruff.toml
Comment (Bryce-3D) [inline ruff.toml:1]: Test

File: ruff.toml (hunk 1 above)
Comment (lunaflight) [reply ruff.toml:7]: Please do, it's a standard codestyle as seen from the official TOML page: https://toml.io/en/

Example:
```toml
[database]
enabled = true
ports = [ 8000, 8001, 8002 ]
data = [ ["delta", "phi"], [3.14] ]
temp_targets = { cpu = 79.5, case = 72.0 }
```

File: .DS_Store
Comment (lunaflight) [inline .DS_Store:1]: Please include this file in `.gitignore` - this should not be tracked.

**Review by lunaflight — CHANGES_REQUESTED** [review]
Just a minor nitpick before we can merge your partial fix on style!

**Comment by lunaflight** [comment]
Looks good to me!

**Comment by Bryce-3D** [comment]
Yay 🧡

**Pull request merged**